css/style.css

```diff
@@ -454,6 +454,11 @@ in the narrow layouts without JS. */
     text-decoration: underline;
 }
 
+.subnav .hide-from-nav
+{
+    display: none;
+}
+
```

Comment:
I couldn't see a way to render the dates only on the changelog listing page and not in the sidebar, hence this CSS hack.

I wish we could do `$($1, ...)`, then we could pass DDoc macro names as parameters.